src/main/java/com/algorand/algosdk/v2/client/algod/PendingTransactionInformation.java

```diff
@@ -9,21 +9,21 @@
 
 
 /**
- * Given a transaction id of a recently submitted transaction, it returns 
- * information about it. There are several cases when this might succeed: 
- * - transaction committed (committed round > 0) - transaction still in the pool 
- * (committed round = 0, pool error = "") - transaction removed from pool due to 
- * error (committed round = 0, pool error != "") 
- * Or the transaction may have happened sufficiently long ago that the node no 
- * longer remembers it, and this will return an error. 
- * /v2/transactions/pending/{txid} 
+ * Given a transaction id of a recently submitted transaction, it returns
+ * information about it. There are several cases when this might succeed:
+ * - transaction committed (committed round > 0) - transaction still in the pool
+ * (committed round = 0, pool error = "") - transaction removed from pool due to
```

Comment:
Looks like we're removing the newlines in the description here.

Reply:
Can you be more specific? Which line is removed? 
It is only removing the space at the end of the line.

Reply:
You can see that the comment is malformed, the bulleted list is missing its newlines.

Reply:
This was like this since a long time ago. I will look into this again. The fix may be in the generator or in the spec file comments. We'll see. 
Also, the comments are not go lint compliant. The names should start with the method name. Will address these later.